src/Components/Forms/FormikInput.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react"
+import { FormikProps, Field } from "formik"
+import styled from "styled-components"
+import { default as Input, InputProps } from "../Input"
+
+// export interface FormikInputProps<T> extends InputProps {}
+
+/**
+ * An input with the standard FormimkProps added for rendering niceness
+ */
+export const FormikInput: (props: any) => any = props => {
```

Comment:
we should name this component `Field` if it's an abstraction for our `Input` component + `Formik`, and use a concrete interface instead of `any` (maybe just reuse `InputProps`)